packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -117,14 +138,36 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
     accessibility: treeBehavior as Accessibility,
   };
 
-  static autoControlledProps = ['activeItemIds'];
+  static autoControlledProps = ['activeItemIds', 'selectedItemIds'];
 
   static Item = TreeItem;
   static Title = TreeTitle;
 
   static getAutoControlledStateFromProps(nextProps: TreeProps, prevState: TreeState) {
-    const { items } = nextProps;
-    let { activeItemIds } = nextProps;
+    const { items, selectable } = nextProps;
+    let { activeItemIds, selectedItemIds } = nextProps;
+
+    if (selectable) {
```

Comment:
why not add here also the `!selectedItemIds && items` check?

Reply:
thank you for catch, definitely it should be there :)